testing/e2e/skr-tester/pkg/keb/client.go

```diff
@@ -0,0 +1,341 @@
+package keb
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+const (
+	SCOPES                     = "broker:write"
```

Comment:
It is just one scope. Not sure how to name it.

Reply:
Changed to `scope`.